src/app/MenuView.ts

```diff
@@ -14,92 +15,63 @@ export class MenuView {
 
   displayMenu(): void {
     Render.removeAllChildren('#sf-app');
-    Render.render(
-      '#sf-app',
-      this.createLandingPage(),
-      this.createFooter(),
-    );
+    Render.render('#sf-app', this.createLandingPage());
   }
 
   private createLandingPage(): HTMLElement {
     return Render.elementFactory(
       'div',
       {
-        className: 'page__container menu-window',
+        className: 'page__container',
```

Comment:
I wonder if we would remove the page container that holds menu window, if the layout wouldnt work. Please check, it seems it may not be necessary